### ISSUE TITLE
Version 0.7.0 is not published to Maven Central yet

### ISSUE THREAD

**Issue by Timwun** [Kotlin/kotlinx-kover#384]
Hi, the latest version 0.7.0 isn't published to Maven Central yet.

https://mvnrepository.com/artifact/org.jetbrains.kotlinx/kover
https://central.sonatype.com/artifact/org.jetbrains.kotlinx/kover/0.6.1/versions

Please publish it. Thanks a lot in advance!

**Comment by shanshin** [comment]
Hi,
version `0.7.0` already published to Maven Central, however, the artifact name [has been changed](https://github.com/Kotlin/kotlinx-kover/blob/main/docs/gradle-plugin/migrations/migration-to-0.7.0.md#rename-dependency-in-buildsrc)

You may find the latest versions by the corresponding name https://mvnrepository.com/artifact/org.jetbrains.kotlinx/kover-gradle-plugin

**Comment by shanshin** [comment]
If you would like to keep track all version of the plugin versions with ID `org.jetbrains.kotlinx.kover`, you may use the following link
https://mvnrepository.com/artifact/org.jetbrains.kotlinx.kover/org.jetbrains.kotlinx.kover.gradle.plugin

**Comment by Timwun** [comment]
Thanks a lot. I changed the entry in my buildscript dependencies classpath. Unfortunately now I get an error while with the application of Kover. :/ 
![image](https://github.com/Kotlin/kotlinx-kover/assets/13067978/5539c1d6-8184-432d-9186-9655eb33b890)

**Comment by shanshin** [comment]
Could you provide a complete example of the build script?

**Comment by Timwun** [comment]
`import org.jetbrains.kotlin.gradle.tasks.KotlinCompile

buildscript {
    repositories {
        mavenCentral()
    }

    dependencies {
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
        classpath "org.jetbrains.kotlinx:kover-gradle-plugin:$kover_version"
    }
}

apply plugin: 'kotlin'
apply plugin: 'kover'

allprojects {
    apply plugin: 'kotlin'
    apply plugin: 'kover'

    repositories {
        mavenCentral()
    }

    // Set target Java version
    java {
        sourceCompatibility = JavaVersion.VERSION_17
        targetCompatibility = JavaVersion.VERSION_17
    }

    // Set Kotlin compiler arguments and Java version
    tasks.withType(KotlinCompile).all {
        kotlinOptions.freeCompilerArgs += ["-opt-in=kotlin.RequiresOptIn"]
        kotlinOptions.allWarningsAsErrors = false
        kotlinOptions.jvmTarget = 17
    }
}`

**Comment by shanshin** [comment]
Please try `apply plugin: 'org.jetbrains.kotlinx.kover'` insted of `apply plugin: 'kover'`

**Comment by Timwun** [comment]
That worked. Now I get another problem with koverXmlReport. :/

**Comment by shanshin** [comment]
Please refer to [migration guide](https://github.com/Kotlin/kotlinx-kover/blob/main/docs/gradle-plugin/migrations/migration-to-0.7.0.md#rename-dependency-in-buildsrc). DSL has been significantly redesigned compared to version 0.6.x, so review the basic migration steps.

You will need to rework the code by moving all settings into `kover` and `koverReport` blocks

**Comment by byates-fleetian** [comment]
The installation documentation is in need of an update to reference this migration.

**Comment by shanshin** [comment]
Yes, thanks.
It is also worth to fix the example with `apply plugin: 'kover'`

**Comment by shanshin** [comment]
Try to use `finalizedBy("koverXmlReport")` instead of `finalizedBy(koverXmlReport)`

**Comment by shanshin** [comment]
I would not recommend adding such a dependency  `finalizedBy("koverXmlReport")`, because in this case, running the test will always result in generating a report, even if it is not necessary.
In Gradle, it is preferable to get different results by calling different commands and passing different parameters.

**Comment by Timwun** [comment]
Good point! :) I will check that out for sure. Thanks for the help. I will close this issue as it got solved.
